.github/workflows/daily.yml

```diff
@@ -76,7 +76,7 @@ jobs:
     if: |
       (github.event_name == 'workflow_dispatch' || (github.event_name != 'workflow_dispatch' && github.repository == 'redis/redis')) &&
       !contains(github.event.inputs.skipjobs, 'fortify')
-    container: ubuntu:lunar
+    container: ubuntu:plucky
```

Comment:
@vitahlin can you try `ubuntu:latest`?
the reason we used lunar is that the latest ubuntu didn't support gcc13 at that time.